src/index.ts

```diff
@@ -344,6 +353,11 @@ class Marian {
                 assert.ok(manifest.manifestRevisionId)
 
                 await session.withTransaction(async () => {
+                    await this.manifests.updateOne(
```

Comment:
will this PR change the current workflow (deploying, triggers creating new manifest, poking marian so that it refreshes)? or will that still be workflow

Reply:
Workflow will be unchanged. We will likely want to replace search-transport with a Realm app at some point in the future, but this is a stopgap to that future goal.